cpp/include/cudf/strings/convert/convert_datetime.hpp

```diff
@@ -84,7 +84,7 @@ std::unique_ptr<column> to_timestamps(
  * | :-------: | ----------- |
  * | \%d | Day of the month: 01-31 |
  * | \%m | Month of the year: 01-12 |
- * | \%y | Year without century: 00-99 |
+ * | \%y | Year without century: 00-99. [0,68] maps to [2000,2068] and [69,99] maps to [1969,1999] |
```

Comment:
We could add a `%C` format string to specify the century like `std::chrono::parse`: https://en.cppreference.com/w/cpp/chrono/parse

Reply:
Reference #5991